projects/fine_tuning/toolbox/fine_tuning_run_fine_tuning_job/templates/retrieval_pod.yaml.j2

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+ name: fine-tuning-file-retrieval
+spec:
```

Comment:
you need the `namespace: {{ ... }}` here
without it it will work locally ... but fail in the CI. Too easy to get bitten by this one :D